File: sandbox/fixtures/javascript/hello.js
/**
 * A simple greeting module.
 */

function greet_user(name) {
  return `Hello, ${name}!`;
}

function add(a, b) {
  return a + b;
}

function subtract(a, b) {
  return a - b;
}

module.exports = { greet, add, subtract };

if (require.main === module) {
  console.log(greet('CogniCode'));
  console.log(`2 + 3 = ${add(2, 3)}`);
}

File: sandbox/fixtures/go-hello/hello.go
// Package main provides a simple example Go module for CogniCode sandbox testing.
package main

import "fmt"

// Greet returns a greeting string for the given name.
func Greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

// Add adds two integers and returns the result.
func Add(a, b int) int {
	return a + b
}

// Subtract subtracts b from a.
func Subtract(a, b int) int {
	return a - b
}

// Multiply multiplies two integers (used for mutation testing only).
func Multiply(a, b int) int {
	return a * b
}

func main() {
	fmt.Println(Greet("CogniCode"))
	fmt.Printf("2 + 3 = %d\n", Add(2, 3))
}
